File: python3/fizzbuzz.py
# leetcode

class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        ans = []
        
        for i in range(1, n+1):
            word = ''
            
            if i % 3 == 0:
                word += 'Fizz'
            if i % 5 == 0:
                word += "Buzz"
            if word == '':
                word = str(i)
            ans.append(word)
        
        print(ans)
        return ans

File: golang/fizzbuzz.go
package main

// Space   : O(n)
// Time    : O(n)

import (
	"strconv"
)

func fizzBuzz(n int) []string {
	var ans []string
	var item string

	for i := 0; i < n; i++ {
		item = ""
		if (i+1)%3 == 0 {
			item += "Fizz"
		}
		if (i+1)%5 == 0 {
			item += "Buzz"
		}
		if item == "" {
			item = strconv.Itoa(i + 1)
		}
		ans = append(ans, item)
	}

	return ans
}
